packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -3920,7 +3925,48 @@ export class SnapController extends BaseController<
   }
 
   /**
-   * Updates the permissions for a snap following an install, update or rollback.
+   * Get the permissions to grant to a Snap following an install, update or
+   * rollback.
+   *
+   * @param snapId - The snap ID.
+   * @param newPermissions - The new permissions to be granted.
+   * @returns The permissions to grant to the Snap.
+   */
+  #getPermissionsToGrant(snapId: SnapId, newPermissions: RequestedPermissions) {
```

Comment:
Do we still need this? I thought we were making changes elsewhere for the auto granting?

Reply:
We grant permission when calling `wallet_switchEthereumChain`, but still need to grant permission for some network when first installing or updating the Snap.

Reply:
Makes sense! Can we document this?